### PR TITLE
kolobova2week

### PR DESCRIPTION
**Pull request by VeronikaKolobova**

### CODE REVIEW

File: force-app/main/default/classes/tasks/BasicTasks.cls [hunk 1]
```diff
@@ -24,7 +33,19 @@ public class BasicTasks {
   public static Decimal mathPower(Integer a, Integer n) {
     //write your code below this line
     Decimal result = 0;
-
+    Decimal c = 0;
+    Decimal d = 1;
+    if (n == 0) {
```
Comment (nchursin) [inline force-app/main/default/classes/tasks/BasicTasks.cls]: Can you do it without this `if`?

**Review by nchursin — COMMENTED** [review]
One remark, eveyrthing else looks good 👍

**Review by nchursin — CHANGES_REQUESTED** [review]
Please fix the remark, and that'll be it

File: force-app/main/default/classes/tasks/BasicTasks.cls (hunk 1 above)
Comment (VeronikaKolobova) [reply force-app/main/default/classes/tasks/BasicTasks.cls]: Hey, updated the file. set the results for 1 be default and the loop won't start unless n>0

File: force-app/main/default/classes/tasks/BasicTasks.cls [hunk 2]
```diff
@@ -33,18 +33,14 @@ public class BasicTasks {
   public static Decimal mathPower(Integer a, Integer n) {
     //write your code below this line
     Decimal result = 0;
-    Decimal c = 0;
+    Decimal c = 1;
     Decimal d = 1;
-    if (n == 0) {
```
Comment (VeronikaKolobova) [inline force-app/main/default/classes/tasks/BasicTasks.cls]: delete this line

File: force-app/main/default/classes/tasks/BasicTasks.cls [hunk 3]
```diff
@@ -3,7 +3,7 @@ public class BasicTasks {
   public static Long sumIntegers(Integer a, Integer b) {
     // ignore public static.. Focus on variables
     //write your code below this line
-    Integer sum = 0;
+    Long sum = a + b;
```
Comment (nchursin) [inline force-app/main/default/classes/tasks/BasicTasks.cls:6]: I just double-checked this task. There's an issue to be fixed. Try running in the terminal to see the test results. Everything else looks good

```
sfdx force:apex:test:run -t CollectionsTasksTest,BasicTasksTest -w 10
```

File: force-app/main/default/classes/tasks/BasicTasks.cls [hunk 4]
```diff
@@ -3,7 +3,7 @@ public class BasicTasks {
   public static Long sumIntegers(Integer a, Integer b) {
     // ignore public static.. Focus on variables
     //write your code below this line
-    Integer sum = 0;
```
Comment (VeronikaKolobova) [inline force-app/main/default/classes/tasks/BasicTasks.cls:6]: need to delete the line 'Integer sum = 0'

**Review by nchursin — APPROVED** [review]
DONE!

**Pull request closed without merging**